lightkurve/periodogram.py

```diff
@@ -157,6 +157,25 @@ def smooth(self, method='boxkernel', filter_width=0.1):
         where `filter width` is in log10(frequency) space. This is best for
         estimating the noise background, as it filters over the seismic peaks.
 
+        Periodograms that are unsmoothed have multiplicative noise that is
+        distributed as chi squared 2 degrees of freedom.  This noise
+        distirbution has a well defined mean and median but the two are not
+        equivalent.  The mean of a chi squared 2 dof distribution is 2, but the
+        median is 2(8/9)**3.
+        (see https://en.wikipedia.org/wiki/Chi-squared_distribution)
+        In order to maintain consistency between 'boxkernel' and 'logmedian' a
```

Comment:
Am I understanding correctly that `boxkernel` is then a 'mean' smoothing, not a 'median' filter?

Reply:
Yes - the boxkernel is like a running mean (in fact I think it is a running mean).